### ISSUE TITLE
Unable to collapse export panel

### ISSUE THREAD

**Issue by iwehrman** [adobe-photoshop/spaces-design#2050]
```
Uncaught TypeError: Cannot read property 'parentElement' of null PanelSet.jsx:156 
```

**Comment by iwehrman** [comment]
CC @designedbyscience @mcilroyc. I suspect @designedbyscience knows how to fix this since he fixed a similar problem previously with the Libraries panel.

**Comment by designedbyscience** [comment]
It needs `ref={UI.EXPORT_PANEL}` in PanelSet.jsx

**Comment by mcilroyc** [comment]
I implemented eric's suggestion in PR #2056 and it *helps*, but there is still some work to be done.  Lucky for me, he volunteered, so assigning this over...
